### PR TITLE
CSharp submission

### PR DESCRIPTION
**Pull request by thainguyentran**

### CODE REVIEW

File: week0002/thainguyentran/CSharp/week0002.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                collum++;
+                posX--;
+                posY--;
+            }
+            for (int i = 0; i < N; i++)
+            {
+                for (int j = 0; j < M; j++)
+                {
+                    if (spiral[i, j] < 10)
+                        Console.Write("  {0}", spiral[i, j]);
```
Comment (vietlq) [inline week0002/thainguyentran/CSharp/week0002.cs]: If it goes up to `100` ;)

File: week0002/thainguyentran/CSharp/week0002.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                {
+                    spiral[posX, i] = value++;
+                }
+                //going up
+                for (int j = posX - 1; j >= row + 1; j--)
+                {
+                    spiral[j, collum] = value++;
+                }
+                row++;
+                collum++;
```
Comment (vietlq) [inline week0002/thainguyentran/CSharp/week0002.cs]: `column`

File: week0002/thainguyentran/CSharp/week0002.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
+using System;
+
+namespace DVRK
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Enter N: ");
```
Comment (vietlq) [inline week0002/thainguyentran/CSharp/week0002.cs]: This is good! I like it!

Comment (vietlq) [inline week0002/thainguyentran/CSharp/week0002.cs]: Btw, what is N and what is M? For better UX, try this:

```
Enter the number of rows M:
4
Enter the number of columns N:
4
```

**Comment by thainguyentran** [comment]
updated the UX

**Comment by vietlq** [comment]
Awesome! Thank you! Now let's move to the week 3!

**Pull request merged**